Quantum-Control-Applications/Superconducting/Multiple Flux Tunable Transmons/Use Case 2 - Two Qubit Randomized Benchmarking/README.md

```diff
@@ -0,0 +1,238 @@
+```
+
+To plot the fidelity for increasing circuit depth you can run the function
+
+```python
+res.plot_fidelity()
+```
+
+----------------------------------------------------------------------------------------------------------------------------------------------------------
+# Additional Information
+
+## Implementation in QUA
+The python program *two_qubit_rb_example.py* implements Two-Qubit Randomized Benchmarking with the described setup and the *TwoQubitRb* class. The decomposition of the two-qubit unitaries into CZ and single qubit gates is given in Ref. [^5]. The circuit generation is done using the *baking* tool from the *py-qua-tools* library. Randomization is done prior to the execution using tableau calculation, also to find the inverse operation. The sequences are passed to the OPX using *input stream*.
+
+[^5]: Barends, R. et al. Nature 508, 500–503 (2014)
+
+### The TwoQubitRB Class
+```python
+rb = TwoQubitRb(config, single_qubit_gate_generator, two_qubit_gate_generators, prep_func, measure_func, verify_generation=True)
+```
+
+- **TwoQubitRb**: The class for generating the configuration and running two-qubit randomized benchmarking experiments with the OPX
+- **config**: dict – Standard configuration “config” containing the relevant experimental details (e.g. what analog outputs are connected to the xy drive, z flux line, etc.).
+- **single_qubit_gate_generator**: A callable used to generate a generic (baked) single qubit gate using a signature similar to phasedXZ
+- **two_qubit_gate_generators**: Mapping two qubit gate names to callables used to generate the (baked) gates (needs at least one two-qubit gate). Can contain all two-qubit gates implemented by the user.
+- **prep_func**: Callable used to reset the qubits to the |00> state. This function does not use the baking object, and is a proper QUA code macro (e.g. wait() statement or active reset protocol).
+- **measure_func**: A callable used to measure the qubits. This function does not use the baking object, and is a proper QUA code macro. Returns a tuple containing the measured values of the two qubits as QUA expressions. The expression must evaluate to a Boolean value. False means |0>, True means |1>. The most significant bit (MSB) is the first qubit. 
+verify_generation: bool = False
+
+### Run the 2 QB RB 
+The experiment is run by calling the run method of the previously generated program rb.
+
+```python
+qmm = QuantumMachinesManager('127.0.0.1',8080)
+res = rb.run(qmm, circuit_depths=[1, 2, 3, 4, 5], num_circuits_per_depth=50, num_shots_per_circuit=1000)
+```
+
+For running the experiment the user has to specify the following arguments:
+- **qmm**: The quantum machine manager instance, on which the 2-Qubit-RB will be executed on.
+- **circuit_depths**: Number of consecutive clifford gates (layers) per sequence (not including the inverse.
+- **num_circuits_per_depth**: The amount of different circuit randomizations (combination of Cliffords) in each sequence. 
```

Comment:
Maybe specify that the results from these random sequences will be averaged together (that's the case right?)